packages/angular/cli/utilities/json-schema.ts

```diff
@@ -19,16 +19,20 @@ import {
   Value,
 } from '../models/interface';
 
-function _getEnumFromValue<E, T extends string>(v: json.JsonValue, e: E, d: T): T {
-  if (typeof v !== 'string') {
-    return d;
+function _getEnumFromValue<E, T = E[keyof E]>(
```

Comment:
The types here had to be changed due to TS 3.1, I updated the param names to make the more readable.